velox/docs/functions/presto/string.rst

```diff
@@ -255,6 +255,14 @@ String Functions
 
     Converts ``string`` to uppercase.
 
+.. function:: word_stem(word) -> varchar
+
+    Returns the stem of ``word`` in the English language.
+
+.. function:: word_stem(word, lang) -> varchar
+
+    Returns the stem of ``word`` in the ``lang`` language.
```

Comment:
What are valid values for 'lang'? What happens if 'lang' is not valid?

Reply:
throws an invalid arg exception

Reply:
Please, update the documentation to clarify.

Also, what are valid values for 'lang'?

Reply:
sure. the content is from Presto Java: http://prestodb.io/docs/0.285.1/functions/string.html. no valid values for lang there. but I certainly can add that in here.

Reply:
Presto docs are often lacking important details. Let's make sure Velox docs are more complete.